js/modules/loki_message_api.js

```diff
@@ -77,6 +78,48 @@ class LokiMessageAPI {
   async sendMessage(pubKey, data, messageTimeStamp, ttl, options = {}) {
     const { isPing = false, numConnections = DEFAULT_CONNECTIONS } = options;
     // Data required to identify a message in a conversation
+
+    if (pubKey.match(/^06/)) {
+      // console.log('HIJACK', data);
```

Comment:
Can you remove all these commented out snippets pls

Reply:
Not while it's a WIP